test/openAPI/features/support/helpers/helpers.js

```diff
@@ -252,4 +252,26 @@ module.exports = {
       BirthCertificateID: { type: 'string' },
     },
   },
+  dataListReadEndpoint: 'data/{registryname}/{versionnumber}',
+  dataListResponseSchema: {
+    type: 'object',
+    properties: {
+      count: { type: 'number' },
+      next: { type: 'string' },
+      previous: { type: ['string', 'null'] },
+      results: {
+        type: 'array',
+          items: {
+          type: 'object',
+            properties: {
+            ID: { type: 'string' },
+            FirstName: { type: 'string' },
+            LastName: { type: 'string' },
+            BirthCertificateID: { type: 'string' }
+          }
```

Comment:
This object is also used in another response schema. I suggest to extract this part, named it `exampleSchema` because this is how it's named in API spec and then in result object write `this.exampleSchema`

Reply:
ok